README.md

```diff
@@ -15,7 +15,8 @@ Create Pull Request action will:
    - tracked (modified) files
    - commits made during the workflow that have not been pushed
 2. Commit all changes to a new branch, or update an existing pull request branch.
-3. Create a pull request to merge the new branch into the base&mdash;the branch checked out in the workflow.
+3. Create a pull request to merge the new branch into the base&mdash;the branch checked out in the workflow&mdash;,
```

Comment:
I would prefer if this was more concise. There is a longer explanation of the [action behaviour](https://github.com/peter-evans/create-pull-request#action-behaviour) later in the readme. 
```suggestion
3. Create or update a pull request to merge the branch into the base&mdash;the branch checked out in the workflow.
```